pages.es/common/aireplay-ng.md

```diff
@@ -0,0 +1,9 @@
+# aireplay-ng
+
+> Inyecta paquetes en una red inalámbrica.
+> Parte de `aircrack-ng`.
+> Más información: <https://www.aircrack-ng.org/doku.php?id=aireplay-ng>.
+
+- Envía una cantidad específica de paquetes disociados dada la dirección MAC de un punto de acceso, la dirección MAC de un cliente y una interfaz:
+
+`sudo aireplay-ng --deauth {{count}} --bssid {{ap_mac}} --dmac {{client_mac}} {{interface}}`
```

Comment:
```suggestion
`sudo aireplay-ng --deauth {{1..infinity}} --bssid {{ap_mac}} --dmac {{client_mac}} {{interface}}`
```

Optional fix: [range syntax](https://github.com/tldr-pages/tldr/pull/9646) can be used.